entities/entities/spawned_shipment/init.lua

```diff
@@ -108,6 +108,25 @@ function ENT:Use(activator, caller)
     end)
 end
 
+local function calculateAmmo(class, shipment)
+    local clip1, ammoadd = shipment.clip1, shipment.ammoadd
+
+    local defaultClip, clipSize
+    local wep_tbl = weapons.Get(class)
+    if wep_tbl and wep_tbl.Primary then
+        defaultClip = wep_tbl.Primary.DefaultClip
+        clipSize = wep_tbl.Primary.ClipSize
+    end
+    ammoadd = ammoadd or defaultClip
+
+    -- If the clip is empty, fill it with additional bullets
+    if not clip1 then
```

Comment:
This condition does not do what the comment says it does. The comment mentions empty, but the condition is about `nil` or `false`. In Lua, `not 0` evaluates to `false`.

Reply:
I meant to specify if it was undefined, define it to a full clip (existing behaviour), but remove the extra ammo.